app/services/invite_schools.rb

```diff
@@ -1,6 +1,18 @@
 # frozen_string_literal: true
 
 class InviteSchools
+  def resend_school_invitation(school)
```

Comment:
I don't think we need this at all. You can just call run with [urn]

Reply:
I actually would like to keep it as its specific to only emailing one while the other is used for series. Let me know your opinion

Reply:
If you keep it then you will have to add tests for it and it just adds to the maintenance burden and increases duplication for no added benefit.

Reply:
There is already cypress test for it that tests if the email was sent

Reply:
I still would like to keep it though